android-components/plugins/dependencies/src/main/java/DependenciesPlugin.kt

```diff
@@ -192,6 +192,7 @@ object ComponentsDependencies {
     val mozilla_sync_manager = "${ApplicationServicesConfig.groupId}:syncmanager:${ApplicationServicesConfig.version}"
     val mozilla_push = "${ApplicationServicesConfig.groupId}:push:${ApplicationServicesConfig.version}"
     val mozilla_remote_tabs = "${ApplicationServicesConfig.groupId}:tabs:${ApplicationServicesConfig.version}"
+    val mozilla_suggest = "${ApplicationServicesConfig.groupId}:suggest:${ApplicationServicesConfig.version}"
```

Comment:
Not blocking on this, but curious about the naming convention difference between FxSuggest and MozillaSuggest. Assuming the name here is just following precedent, but it made me wonder if MozillaSuggest would make long-term sense for the code as well

Reply:
Hmmm, I'd almost be tempted to go the other way. "Suggest" is the name of the Application Services component, and the feature is called "Firefox Suggest".

My preferred name would be `mozilla_appservices_suggest`, but none of our other Application Services components use that convention...yet? 😅 WDYT?

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1851213.